tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/monitoring/gcf_ingest_latest_by_table.sql

```diff
@@ -0,0 +1,33 @@
+CREATE OR REPLACE VIEW
```

Comment:
Add Apache 2.0 header / google copyright
```suggestion
/*
 * Copyright 2019 Google LLC
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *     http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

CREATE OR REPLACE VIEW
```